core/functions/serialization.go

```diff
@@ -97,6 +99,8 @@ func DeserializeFunction(ctx context.Context, data []byte) (Function, error) {
 	if version == 1 {
 		f.ExtensionName = reader.String()
 		f.ExtensionSymbol = reader.String()
+		f.SQLDefinition = reader.String()
```

Comment:
These are breaking changes, so we have to add a version 2 and read these only in this. For `if version == 1`, we can change that to `version >= 1` and then add a new `if version == 2` under it.